guides/release/code-editors/index.md

```diff
@@ -53,7 +57,12 @@ An Intellisense engine which takes control over all linting, hinting, and langua
 
 [coc-ember](https://github.com/NullVoxPopuli/coc-ember) -
 Ember.js language server extension including useful configuration instructions. 
-Optional addons for Ember language server:
+
+[nvim-treesitter](https://github.com/nvim-treesitter/nvim-treesitter) -
```

Comment:
this is _way_ easier to use than the bunch of duct taped together extensions below (vim-ember-hbs, vim-javascript, etc).
One meta package for all highlighting. it's nice.